sys-apps/portage/portage-2.3.24.ebuild

```diff
@@ -99,6 +99,10 @@ python_prepare_all() {
 		sed -e 's:\("--dynamic-deps", \)\("y"\):\1"n":' \
 			-i pym/_emerge/create_depgraph_params.py || \
 			die "failed to patch create_depgraph_params.py"
+
+		einfo "Enabling additional FEATURES for gentoo-dev..."
+		echo 'FEATURES+=" ipc-sandbox network-sandbox strict-keepdir"' \
```

Comment:
The make.globals parser doesn't support the += operator so you have to use `FEATURES="${FEATURES}  ..."